plugins/HHAnalyzer.cc

```diff
@@ -238,6 +238,8 @@ void HHAnalyzer::analyze(const edm::Event& event, const edm::EventSetup&, const
             ll.push_back(dilep); 
         }
     }
+    // have the ll collection sorted by ht
+    std::sort(ll.begin(), ll.end(), [&](HH::Dijet& a, HH::Dijet& b){return a.p4.Pt() > b.p4.Pt();});
```

Comment:
fixed: I did mean ht and implemented pt instead, my bad... storing ht is not a bad idea either, I implemented it as well :)

for the `Dilepton` comment: already spotted and fixed in https://github.com/cp3-llbb/HHAnalysis/pull/101/commits/3b52f2d3d425e1efce982fa8c39c2d5241fd49b6#diff-6cdb42586ea6220d1c53fa646556a54cL242